account-kit/react/src/hooks/useSolanaTransaction.ts

```diff
@@ -84,13 +92,23 @@ mutate({
 export function useSolanaTransaction(
   opts: SolanaTransactionHookParams
 ): SolanaTransaction {
-  const web3Context = useContext(AlchemySolanaWeb3Context);
-  const fallbackSigner = useSolanaSigner({});
+  const { config } = useAlchemyAccountContext();
+  const fallbackSigner: void | SolanaSigner = useSolanaSigner({});
```

Comment:
why `void | Signer` here? isn't it `null | Signer` or `undefined | Signer`? I ask because we don't do this anywhere else in the SDK. If in practice all `void` and `undefined` are the same, I'd opt for the latter since it's more consistent

Reply:
also curious why this has to be `useSolanaSigner({})` instead of `useSolanaSigner()`?

Reply:
Okay, thanks. Added default to useSolanaSigner